index.html

```diff
@@ -16,7 +16,12 @@
         <!--header section-->
         <nav class="navigation_bar">
             <span class="my_logo">My Logo</span>
-            <img class="social_media_icons" src="icons/Union.png" alt="menu bar">
+            <ul class="nav-item">
+                <li class="nav-item-li1"><a href="#works">Portoflio</a></li>
+                <li class="nav-item-li2"><a href="#aboutme">About</a></li>
+                <li class="nav-item-li3"><a href="#contact">Contact</a></li>
+            </ul>
+            <img class="social_media_icons" src="./icons/Union.png" alt="menu bar">
```

Comment:
- [ ] As pointed out by the previous reviewer, the background behind the mobile menu should be blurry when it is open and the close(X) icon should not disappear when a user scrolls down.
You can use `backdrop-filter` css property for the blur. 

### Your Design
![image](https://user-images.githubusercontent.com/43995830/223473962-9afeb2ba-f4a4-41f3-8b6f-1bc6ff92d2b0.png)

### Original Design
![image](https://user-images.githubusercontent.com/62705947/223446505-b4142599-4e03-4d63-8374-09b66d1738f7.png)